### ISSUE TITLE
Error gutter shows up to the right of line numbers instead of to the left

### ISSUE THREAD

**Issue by sgraf812** [kak-lsp/kak-lsp#492]
I've been using commit https://github.com/kak-lsp/kak-lsp/commit/354b46e3cf56f0da35b444941a701ca4c1135aa8 (somewhere between v8 and v9) for some time and finally switched to v9 stable this week. But now I noticed that kak-lsp inserts its error gutter to the right of the line numbers, making for a strange gap most of the time:

![Unbenannt](https://user-images.githubusercontent.com/1151264/121495711-22de9c80-c9da-11eb-8fd4-146e8739fbee.PNG)

Note how the error gutter appears to the right, but the git gutter appears to the left.

Previous versions looked like this:

![Unbenannt](https://user-images.githubusercontent.com/1151264/121495884-443f8880-c9da-11eb-9d4d-f91743c11e09.PNG)

with the error gutter directly after the git gutter (not shown in that image).

**Comment by krobelus** [comment]
I think this depends on whichever highlighter is defined first.
If I run

```
lsp-enable
add-highlighter global/ number-lines -hlcursor
```

diagnostics are left of the line numbers etc.
I don't think anything in kak-lsp changed but I could be wrong.

**Comment by sgraf812** [comment]
Hmm. I'm enabling the number-lines highlighter through a nix+home-manager built configuration, so I can't (or rather don't wan to) delay it manually.

I noticed that I also ran lsp-enable immediately after plugin load. I replaced it with a filetype specific hook that calls lsp-enable-window now. That somehow moved the error gutter to the left again.

**Comment by krobelus** [comment]
Yeah, it can be a bit inconvenient to have to take care of this in the user configuration. Maybe there is a better way.

Your configuration system should allow you to choose the order of loading plugins/kakrc.
I load plugins before the rest of my kakrc file. This allows me to override some plugin behavior.

> I replaced it with a filetype specific hook that calls lsp-enable-window now. That somehow moved the error gutter to the left again.

Probably because then the `WinSetOption filetype=haskell` hook from `rc/filetype/haskell.kak` is fired before your LSP hook.
That seems like a good solution.
